gateway/gateway.go

```diff
@@ -268,7 +268,14 @@ func (gw *Gateway) handleMessage(msg config.Message, dest *bridge.Bridge) []*BrM
 
 	// only relay topic change when configured
 	if msg.Event == config.EventTopicChange && !gw.Bridges[dest.Account].GetBool("ShowTopicChange") {
```

Comment:
I am still somewhat confused by the logic here. Mostly due to it not being fully clear to me yet what we are doing, w.r.t. configuration options.  Would it be possible, with the conversation that we have already had to update the PRs description with what the aim is?

In any case the logic here as it is is has an unnecessary duplicate in the check of `ShowTopicChange`.

Reply:
Ok, I think I simplified this: now it sends if there's any topic-related processing features enabled on the other end.